### ISSUE TITLE
Pin configuration

### ISSUE THREAD

**Issue by lukebelz** [gpxlBen/VESC_Logger#1]
So the pin configuration is slightly off (by 1. I'm soldering straight to the vesc, so I noticed this, but for those using the port and plug method, they will be soldering to the wrong pins. here's what the UART port pins are:

pin 1: ADC2
pin 2: Rx
pin 3: Tx
pin 4: ADC
pin 5: GND
pin 6: 3.3v
pin 7: 5v

I got these straight off of one of my vescs.

VCC is not a pin, at least anymore.

This is a vesc v4.11. I don't think the uart port has changed, is this the right config then?

VESC Pin 2 (not 1) (RX) -> Adafruit TX
VESC Pin 3 (not 2) (TX) -> Adafruit RX
VESC Pin 5 (not 4) (GND) -> Adafruit GND + Adafruit CTS
VESC Pin 6 (not 5) (VCC) -> Adafruit vIN

I guess the better question is VCC the 3.3v or 5v?

**Comment by gpxlBen** [comment]
Hi Evoheyax.  I think the pin configuration did change after 4.7, but I can't find the info anywhere now...

Maybe some clues over here:  http://vedder.se/2015/08/vesc-writing-custom-applications/

And looking through Vedders source code too?

You'll have to determine 3.3v or 5v with a multimeter if you're not sure!

**Comment by lukebelz** [comment]
I think your right, it did change. Because that diagram shows what you listed. If you can figure out when it changed, maybe add the pins for different versions? I think most will be using 4.10, 4.11, or 4.12, since that's what most of enertion and ollin have made, which seem to be 2 of the bigger sources right now for vescs. And those 3 versions use the same firmware. So My assumption is that those should have the same pin configuration or else they would need different firmwares.

Basically, my question is do you know if the Adafruit vIN wants 3.3v or 5v? My vesc pcb is labeled so  know which is which, I just don't know what hte Adafruit wants...

Btw, thank you for this work. I plan to add onto this app when I get a chance.
